### PR TITLE
fix json serialization in admissionConfig

### PR DESCRIPTION
**Pull request by deads2k**
Missing `json` tag in master-config.yaml file.  Also updated the unit tests that were intended to catch these conditions.

@csrwng ptal.

### CODE REVIEW

**Comment by deads2k** [comment]
[test]

File: pkg/api/install/tags_test.go [hunk 1]
```diff
@@ -96,7 +106,46 @@ func checkJsonTags(objType reflect.Type, seen *map[reflect.Type]bool, t *testing
 
 		switch structField.Type.Kind() {
```
Comment (csrwng) [inline pkg/api/install/tags_test.go]: should this handle pointers to structs ?

**Comment by csrwng** [comment]
Just one question, otherwise LGTM

File: pkg/api/install/tags_test.go (hunk 1 above)
Comment (deads2k) [inline pkg/api/install/tags_test.go]: > should this handle pointers to structs ?

Updated

**Comment by deads2k** [comment]
[merge]

**Pull request merged**